lightbulb/app.py

```diff
@@ -344,6 +344,7 @@ async def _manage_application_commands(self, _: hikari.StartingEvent) -> None:
 
         try:
             await internal.manage_application_commands(self)
+            await self.dispatch(events.LightbulbStartedEvent(app=self))
```

Comment:
This should be outside of the `try:` block because it would not be called if a `ForbiddenError` was raised. Put it at the top of the else block.